packages/dapp/src/hooks/useBlockchainExplorerURL.ts

```diff
@@ -3,11 +3,11 @@ import { useWallet } from '../providers';
 
 export const useBlockchainExplorerURL = () => {
   const { account } = useWallet();
-  const { getChainById } = useChains();
+  const { isSuccess, getChainById } = useChains();
```

Comment:
can we make this bit more readable by renaming isSuccess in this file?
`const {isSuccess: chainsLoaded, getChainById} = useChains();`